src/utils.js

```diff
@@ -27,11 +27,14 @@ const exec = async (command) => {
 
 const mimeType = async (opts) => {
   const { path, buffer } = opts;
-  FileMagic.magicFile = PATH.join(
-    __dirname,
-    '..',
-    'node_modules/@npcz/magic/dist/magic.mgc'
-  );
+  let magicFile = PATH.join(__dirname, '../assets/magic.mgc');
+  if (!fs.existsSync(magicFile))
```

Comment:
“[_No sync functions. Please use async_](https://github.com/iterative/cml/pull/391#discussion_r574623597)” 😉 I guess that, in this specific case, using a synchronous function is easier than [using a deprecated one](https://nodejs.org/api/fs.html#fsexistspath-callback), though.

Reply:
Yes, we have a couple of sync in the code due to not very nice async counterparts and the side effect is not horrible due to the nature of the command (just only publish one file) VS a more demanding command like the runner